Vagrantfile

```diff
@@ -52,29 +54,41 @@ Vagrant.configure(2) do |config|
         end
     end
 
-    # system setup for patroni ndoes
+    # system setup for patroni nodes
     (patroni_nodes).each do |node|
         config.vm.define node do |conf|
-            args = [ '-n', node, '-v', pgver ];
+            args_pa = [ '-n', node, '-v', pgver ];
+            args_vm = [ '-n', node, '-v', vmver, '-i', vip_ip.to_s ];
+
```

Comment:
no need for two distinct array. Just put common args in the array, then concat additionnal args later when needed. See bellow for an example

Reply:
I thought it was more readable.
Done as suggested.